packages/hoppscotch-app/components/http/ResponseMeta.vue

```diff
@@ -137,8 +137,10 @@ const props = defineProps<{
 
 const statusCategory = computed(() => {
   if (
-    props.response.type === "loading" ||
-    props.response.type === "network_fail"
+    response.type === "loading" ||
+    response.type === "network_fail" ||
+    response.type === "script_fail" ||
+    response.type === "fail"
```

Comment:
Where is response coming from ? TypeScript is giving an error because it can't resolve it. Make sure you resolve TypeScript errors before commiting.

It should be:
```suggestion
    props.response.type === "loading" ||
    props.response.type === "network_fail" ||
    props.response.type === "script_fail" ||
    props.response.type === "fail"
```

Reply:
Needful done. Resolving.